src/ducks/connections/index.js

```diff
@@ -334,32 +284,6 @@ export const endConnectionCreation = () => ({
   type: END_CONNECTION_CREATION
 })
 
-// action creators async
-export const deleteConnection = trigger => {
```

Comment:
Really happy to remove this function, but are we sure that the account is no more appearing in Account Picker ? I am afraid that we still need to update the redux slice `connection`. Can you check this ?

Reply:
You're right, I didn't see that point, thanks 👍

Reply:
Fixed in f423dc66dc002f901d813a5eb9c06aec36ec4888